src/store.ts

```diff
@@ -0,0 +1,18 @@
+import { configureStore } from "@reduxjs/toolkit"
+
+import rootReducer from "./reducers/rootReducer"
+import createSagaMiddleware from "redux-saga"
+import rootSaga from "./sagas/rootSaga"
+
+const sagaMiddleware = createSagaMiddleware()
+
+const store = configureStore({
+  reducer: rootReducer,
+  middleware: [sagaMiddleware],
```

Comment:
Note that this is leaving out any of the default middleware that RTK comes with, including both the thunk middleware and the mutation checking middleware.  I see that you're using sagas, so thunks may not be as critical, but it's worth keeping the others in there.  To do that, import `getDefaultMiddleware`, and change this line to:

```ts
  middleware: [...getDefaultMiddleware({thunk: false}), sagaMiddleware]
```